### PR TITLE
53 create payment model

### PR DESCRIPTION
**Pull request by qAziz060708**

### CODE REVIEW

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/DTO/RequestDTOs/PaymentRequestDTO.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+namespace PharmacyShopping.BusinessLogic.DTO.RequestDTOs
+{
+    public class PaymentRequestDTO
+    {
+        public int SaleID { get; set; }
+
+        public string CreditCardNumber { get; set; }
+
+        public double TotalPrice { get; set; }
+
```
Comment (isfandiyoro5) [inline OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/DTO/RequestDTOs/PaymentRequestDTO.cs]: line qoshilib qopti

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/DTO/RequestDTOs/PaymentRequestDTO.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿using FluentValidation;
+using PharmacyShopping.BusinessLogic.DTO.RequestDTOs;
+
+namespace PharmacyShopping.BusinessLogic.DTO.RequestDTOs
+{
+    public class PaymentRequestDTO
+    {
+        public int SaleID { get; set; }
```
Comment (isfandiyoro5) [inline OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/DTO/RequestDTOs/PaymentRequestDTO.cs]: SaleID ni d sini kichkina qip qoying

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/DTO/ResponseDTOs/PaymentResponseDTO.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿namespace PharmacyShopping.BusinessLogic.DTO.ResponseDTOs
+{
+    public class PaymentResponseDTO
+    {
+        public int PaymentId { get; set; }
+
+        public int SaleID { get; set; }
```
Comment (isfandiyoro5) [inline OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/DTO/ResponseDTOs/PaymentResponseDTO.cs]: buniyam d sini kichkina qilish kk

File: OnlinePharmacyShopping/PharmacyShopping.DataAccess/Models/Payment.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+﻿namespace PharmacyShopping.DataAccess.Models
+{
+    public class Payment
+    {
+        public int PaymentId { get; set; }
+
+        public int SaleID { get; set; }
```
Comment (isfandiyoro5) [inline OnlinePharmacyShopping/PharmacyShopping.DataAccess/Models/Payment.cs]: d sini kichkina qilish kk

**Review by isfandiyoro5 — CHANGES_REQUESTED** [review]

**Comment by qAziz060708** [comment]
I Fixed changes

**Review by isfandiyoro5 — APPROVED** [review]

**Pull request merged**